sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidOperatorTable.java

```diff
@@ -304,8 +304,10 @@ public class DruidOperatorTable implements SqlOperatorTable
                    .add(new NestedDataOperatorConversions.JsonKeysOperatorConversion())
                    .add(new NestedDataOperatorConversions.JsonPathsOperatorConversion())
                    .add(new NestedDataOperatorConversions.JsonQueryOperatorConversion())
-                   .add(new NestedDataOperatorConversions.JsonValueOperatorConversion())
                    .add(new NestedDataOperatorConversions.JsonValueAnyOperatorConversion())
+                   .add(new NestedDataOperatorConversions.JsonValueBigintOperatorConversion())
+                   .add(new NestedDataOperatorConversions.JsonValueDoubleOperatorConversion())
+                   .add(new NestedDataOperatorConversions.JsonValueStringOperatorConversion())
```

Comment:
Do these need to be registered with the operator table in order for the convertlet to return them? Asking since, if possible, it'd be nice if end users couldn't write these functions directly.

Not a requirement, I suppose. If they have to be here then it's fine.

Reply:
I couldn't figure out a way to make it work, but it feels like something that should be possible....

```
org.apache.druid.sql.calcite.planner.UnsupportedSQLQueryException: Cannot build plan for query. Possible error: SQL query requires 'JSON_VALUE_DOUBLE' operator that is not supported.
``` is what happens if i remove them for a query using `JSON_VALUE`

Reply:
OK, got it. I think it's fine if these functions remain accessible, in that case.